test/mp_correct/src/do20.f90

```diff
@@ -0,0 +1,27 @@
+!* Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+!* See https://llvm.org/LICENSE.txt for license information.
+!* SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+!	parallel do with negative stride
```

Comment:
I suggest mentioning the schedule type, like in the other `pd*.f` and `do*.f` tests:
```fortran
!   simple parallel do, negative stride
```